tsconfig.json

```diff
@@ -57,10 +57,18 @@
     },
     {
       "path": "./tests"
+    },
+    {
+      "path": "./scripts"
     }
   ],
   "ts-node": {
     "esm": true,
-    "experimentalSpecifierResolution": "node"
+    "experimentalSpecifierResolution": "node",
```

Comment:
I thought it could be better to use the same rule for node scripts.

Reply:
I don't think so. For `ts-node` I think unused locals/params checking is unnecessary. In most cases using `ts-node` is for developing or debugger. However, for compile-time, it's somehow a linter (like ESLint)